hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsClient.java

```diff
@@ -875,7 +896,7 @@ public AbfsRestOperation getPathStatus(final String path,
       abfsUriQueryBuilder.addQuery(HttpQueryParams.QUERY_PARAM_ACTION, AbfsHttpConstants.GET_STATUS);
       operation = SASTokenProvider.GET_STATUS_OPERATION;
     } else {
-      addEncryptionKeyRequestHeaders(path, requestHeaders, false, null,
+      addEncryptionKeyRequestHeaders(path, requestHeaders, false, encryptionAdapter,
```

Comment:
Inline with https://github.com/apache/hadoop/pull/3440#pullrequestreview-1133031625. AbfsClient to not heuristic on creating encryptionAdapter.